db/new_seeds/base/add_lead_providers_and_cips.rb

```diff
@@ -87,3 +87,8 @@
 }.each do |cpd_lead_provider, id|
   FactoryBot.create(:seed_npq_lead_provider, cpd_lead_provider:, id:, name: cpd_lead_provider.name)
 end
+
+CpdLeadProvider.find_each do |clp|
+  lp = LeadProvider.find_or_create_by!(name: clp.name)
+  lp.update!(cpd_lead_provider: clp)
```

Comment:
isn't this happening on line 63? as we also have to add cohorts to those lead providers?

Reply:
sorry i didn't meant to commit this, will remove